.travis.yml

```diff
@@ -36,7 +35,7 @@ sudo: required
 
 env:
   global:
-  - CLIPPY_VERS=0.0.156
+  - CLIPPY_VERS=0.0.165
```

Comment:
In my opinion it is better to update clippy version in a separate pull request. I'm not asking you to do this now, but I suppose it is a good practice especially for such huge changes that are hard to review even by themselves.